tests/functional/container/test_container.py

```diff
@@ -540,12 +540,15 @@ def flush_and_check(truncated=False, objects=0, usage=0):
         flush_and_check(truncated=True, objects=16, usage=16384)
         flush_and_check()
 
-    def _check_missing_chunks(self, expected_missing_chunks):
+    def _check_missing_chunks(self, expected_damaged_objects,
```

Comment:
It should be nice to have test with `append` and `truncate` method operations on object and `flush` on container.